framework/include/transfers/MultiAppInterpolationTransfer.h

```diff
@@ -11,9 +11,14 @@
 
 // MOOSE includes
 #include "MultiAppConservativeTransfer.h"
+#include "MooseVariableFEBase.h"
 
 #include "libmesh/mesh_base.h"
 
+#include "libmesh/meshfree_interpolation.h"
+#include "libmesh/system.h"
+#include "libmesh/radial_basis_interpolation.h"
```

Comment:
Looks like you only need a forward declaration of 
```c++
namespace libMesh
{
template <unsigned int>
class InverseDistanceInterpolation;
}
```
and then you can move all these header dependencies to the `.C` file